### PR TITLE
improvement: applications modal + connected state

### PR DESCRIPTION
**Pull request by juliendonck**

### CODE REVIEW

File: src/lib/components/ApplicationDetail/index.svelte [hunk 1]
```diff
@@ -1,13 +1,45 @@
 <script lang="ts">
+	import { get } from 'svelte/store';
+	import { providerStore } from 'web3-stores';
+	import { workstreamStore } from '$lib/stores/workstreamsStore.js';
+	const workstreams = get(workstreamStore);
```
Comment (deckardkrug) [inline src/lib/components/ApplicationDetail/index.svelte:5]: In a svelte component you should be able to subscribe to the store directly by using the `$` sign.

`$workstreamStore`

Comment (juliendonck) [reply src/lib/components/ApplicationDetail/index.svelte:5]: Good point!

File: src/lib/stores/workstreamsStore.js [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import { writable } from 'svelte/store';
 
-export const workstreams = writable([]);
+export const workstreamStore = writable([]);
```
Comment (juliendonck) [inline src/lib/stores/workstreamsStore.js:3]: @deckardkrug naming the store `workstreamStore` helps a lot with readability imo.

Comment (deckardkrug) [reply src/lib/stores/workstreamsStore.js:3]: Okay, I will make that change.

File: src/routes/[id].svelte [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
 	import { get } from 'svelte/store';
-	import { workstreams } from '$lib/stores/workstreamsStore.js';
+	import { workstreamStore } from '$lib/stores/workstreamsStore.js';
 	import { hyphanateString } from '$lib/utils/format';
-	const workstreamsData = get(workstreams);
+	const workstreams = get(workstreamStore);
 
 	export async function load({ params }) {
-		let workstream = workstreamsData.find(
+		let workstream = await workstreams.find(
```
Comment (deckardkrug) [inline src/routes/[id].svelte:8]: Awaiting the `find` method does not seem to be necessary.

File: src/lib/components/ApplicationDetail/index.svelte [hunk 4]
```diff
@@ -1,13 +1,43 @@
...
+
 	export let application;
+	let workstream;
+
+	function getWorkstream() {
+		return (workstream = $workstreamStore.find(
+			(workstream) => workstream.id === application.workstream_id
+		));
+	}
+	getWorkstream();
```
Comment (deckardkrug) [inline src/lib/components/ApplicationDetail/index.svelte:16]: If you want to call `getWorkstream` once the site as loaded, I recommend to do this inside `onMount`.

**Comment by deckardkrug** [comment]
@juliendonck I propose to merge #19 first. As this pr is smaller it will be easier to fix the merge conflicts that my arise from the merge.

I am going to take care of these so you don't have to, as it is my pr that could potentially conflict with yours.

**Review by deckardkrug — APPROVED** [review]

**Pull request merged**